rules/linux/process_creation/proc_creation_lnx_susp_curl_fileupload.yml

```
@@ -21,17 +22,20 @@ logsource:
     category: process_creation
     product: linux
 detection:
-    selection:
+    selection_img:
         Image|endswith: '/curl'
-        CommandLine|contains:
-            - ' -F '
+    selection_cli:
+        - CommandLine|contains:
             - ' --form' # Also covers the "--form-string"
-            - ' -T '
             - ' --upload-file '
-            - ' -d '
             - ' --data '
             - ' --data-' # For flags like: "--data-ascii", "--data-binary", "--data-raw", "--data-urlencode"
-    condition: selection
+        - CommandLine|re: '\s-(F|T|d)\s' # We use regex to ensure a case sensitive argument detection
```

```suggestion
        - CommandLine|re: '\s-[FTd]\s' # We use regex to ensure a case sensitive argument detection
```
minor, but should be the "semantic" and "faster" regex for this job